### PR TITLE
xf86-video-nouveau: update to 1.0.17.

### PR DESCRIPTION
**Pull request by DragonGhost7**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by DragonGhost7** [comment]
I saw it was orphaned, so decided to adopt

File: srcpkgs/xf86-video-nouveau/template [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+revision=1
 build_style=gnu-configure
 hostmakedepends="pkg-config"
 makedepends="eudev-libudev-devel libdrm-devel xorg-server-devel"
 depends="virtual?xserver-abi-video-24_1 mesa-dri"
 short_desc="Xorg opensource NVIDIA video driver"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="DragonGhost7 <darkiridiumghost@gmail.com>"
 license="MIT"
 homepage="http://nouveau.freedesktop.org/wiki/"
```
Comment (ericonr) [inline srcpkgs/xf86-video-nouveau/template:12]:
```suggestion
homepage="http://nouveau.freedesktop.org"
```

Comment (DragonGhost7) [reply srcpkgs/xf86-video-nouveau/template:12]: oh, i see, yes will do

**Comment by DragonGhost7** [comment]
Done

**Pull request merged**